Paymetheus.Decred/Wallet/WalletSeed.cs

```diff
@@ -52,6 +64,16 @@ private static byte[] DecodeUserInput(string userInput, PgpWordList pgpWordList)
             return pgpWordList.Decode(splitInput);
         }
 
+        // Returned array contains the double SHA256 hash.
+        public static byte[] DoubleSha256(byte[] value)
```

Comment:
Ideally this would take a length parameter so we can avoid the copy of the first 32 bytes of the 33 byte value being checked, but this is not performance critical code so i'll let it slide.